apps/sms/index.html

```diff
@@ -128,7 +129,7 @@
 
   <body role="application" class="skin-comms theme-communications">
     <article id="main-wrapper" class="wrapper" data-position='right'>
-      <section id="thread-list" class="panel default-panel" role="region">
+      <section class="panel panel-thread-list" role="region">
```

Comment:
not sure how we going to treat these classes like `panel-thread-list` that only used in js(navigation), not for styling. I remember we applied `js-` prefix for these classes. So 'panel-' prefix here also imply this class is not for styling but for navigation?